storage-hbase/src/main/java/org/apache/kylin/storage/hbase/steps/HBaseJobSteps.java

```diff
@@ -63,6 +64,13 @@ public HadoopShellExecutable createCreateHTableStep(String jobId, CuboidModeEnum
         appendExecCmdParameters(cmd, BatchConstants.ARG_SEGMENT_ID, seg.getUuid());
         appendExecCmdParameters(cmd, BatchConstants.ARG_PARTITION,
                 getRowkeyDistributionOutputPath(jobId) + "/part-r-00000");
+        if(this.seg.getConfig().isHFileDistCP()){
+            String partitionOutputPath = getRealizationRootPath(jobId) + "/rowkey_stats/part-r-00000_hfile";
+            appendExecCmdParameters(cmd, BatchConstants.ARG_PARTITION, partitionOutputPath);
+        }else {
+            appendExecCmdParameters(cmd, BatchConstants.ARG_PARTITION,
```

Comment:
Append paramters twice for `BatchConstants.ARG_PARTITION` .

Reply:
new commit fixed this